pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.black]
+skip-string-normalization=true
+line-length = 88
+exclude = '''
+(
+  /(
+      \.eggs         # exclude a few common directories in the
+    | \.git          # root of the project
+    | \.hg
+    | \.mypy_cache
+    | \.nox
+    | \.tox
+    | \.venv
+    | _build
+    | build
+    | dist
+  )/
+  | python/nav/enterprise/ids\.py     # Autogenerated list of constants
+  | python/nav/smidumps/.*\.py        # Autogenerated MIB dumps
+  | python/nav/etc/geomap/config\.py  # Not actually a Python file
```

Comment:
Why is it called `.py` if it is not Python?

Reply:
Because the original author doesn't know how to use their editor. The config file syntax is sufficiently close to Python syntax that Python syntax highlighting is useful, so the author gave the file a `.py` extension to have their editor automatically syntax highlight the file. I think all this is stated directly in the comments of that config file :-P